L/libseat/build_tarballs.jl

```diff
@@ -0,0 +1,49 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libseat"
+version = v"0.5.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://git.sr.ht/~kennylevinsen/seatd/archive/$version.tar.gz", "274b56324fc81ca6002bc1cdd387668dee34a6e1063e5f3896805c3770948988"),
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/seatd-*/
+atomic_patch ../patches/fix_crosscompile.patch
+mkdir build
+cd build
+apk add meson scdoc
+
+# pkgconfig returns the wrong path here. It's an ugly hack, but it works
+mkdir -p /workspace/destdir/usr/bin/
+ln -s $(which scdoc) /workspace/destdir/usr/bin/
```

Comment:
Why do we need to do this also here?  I feel like there is something wrong, but I've never seen this before

Reply:
It was the same issue again. Seems like there is something going wrong with meson and pkgconfig in general.

Reply:
```
sandbox:${WORKSPACE}/srcdir/seatd-0.5.0/build # pkg-config --variable=scdoc scdoc
/workspace/destdir/usr/bin/scdoc
sandbox:${WORKSPACE}/srcdir/seatd-0.5.0/build # pkg-config --path scdoc
/usr/lib/pkgconfig/scdoc.pc
sandbox:${WORKSPACE}/srcdir/seatd-0.5.0/build # cat $(pkg-config --path scdoc)
prefix=/usr
exec_prefix=${prefix}
scdoc=${exec_prefix}/bin/scdoc

Name: scdoc
Description: Man page generator
Version: 1.10.1
```
I don't know, this looks to me like a bug in pkg-config